src/main/resources/META-INF/rewrite/java-version-11.yml

```diff
@@ -196,4 +197,27 @@ recipeList:
       groupPattern: com.sun.xml.ws
       artifactPattern: "*"
       onlyIfVersionsMatch: true
-
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.InternalBindContextFactory
+displayName: Use `com.sun.xml.bind.v2.ContextFactory` instead of `com.sun.xml.internal.bind.v2.ContextFactory`
+description: Do not use the com.sun.xml.internal.bind.v2.ContextFactory class.
```

Comment:
```suggestion
description: Do not use the `com.sun.xml.internal.bind.v2.ContextFactory` class.
```

Reply:
I didn't realize descriptions could use that code syntax. I'll go ahead and forward this to other applicable descriptions.